supp_mat/SM3_ExplainingDeviation.qmd

```diff
@@ -981,13 +981,13 @@ patchwork::wrap_plots(yi_cont_plots,heights = 4, byrow = TRUE) +
 ManyEcoEvo_yi_viz %>%
   filter(exclusion_set == "complete", 
          dataset == "blue tit",
-         model_name %nin% c("MA_mod", "box_cox_rating_cat_no_int")) %>% 
+         model_name %nin% c("box_cox_rating_cont", "box_cox_rating_cat_no_int","sorensen_glm" )) %>% 
   bind_rows({euc_yi_results %>% 
-      filter(model_name %nin% c("MA_mod", "box_cox_rating_cat_no_int"))}) %>% 
+      filter(model_name %nin% c("box_cox_rating_cont", "box_cox_rating_cat_no_int","sorensen_glm", "uni_mixed_effects"))}) %>% 
   mutate( dataset = case_when(str_detect(dataset, "eucalyptus") ~ "Eucalyptus",
                               TRUE ~ dataset),
           model_name = forcats::as_factor(model_name) %>% 
-            forcats::fct_relevel(c("box_cox_rating_cat", 
+            forcats::fct_relevel(c("box_cox_rating_cat_no_int", 
```

Comment:
good pickup

Reply:
Updated to reflect fixes in [aee7c5d](https://github.com/egouldo/ManyAnalysts/pull/66/commits/aee7c5dcc17f09d2366b290bcc5d8db4503a5553)